protos/dcs/atmosphere/atmosphere.proto

```diff
@@ -40,8 +42,10 @@ message GetWindWithTurbulenceRequest {
 message GetWindWithTurbulenceResponse {
   // The heading the wind is coming from.
   float heading = 1;
-  // The strength of the wind in meters per second
+  // The strength of the wind in meters per second.
   float strength = 2;
+  // The direction (it is blowing to) and speed (magnitude of the vector) of the wind.
+  dcs.common.Vector velocity = 3;
```

Comment:
I actually don't need the vector in the response for my own use-case anymore. So I'd be fine with not including it in the response, but I was wondering whether there might still be use-cases for the velocity in 3D space rather then the strength in 2D. We could also remove it again, until the need comes up again. Wdyt?

Reply:
I cannot think of any use-case right now where a client would need this information and since it is the opposite of `heading` (From and To) I would leave it out for the moment.

If we leave it in then we should add more documentation around these two values and why they are opposite.

Reply:
Removed it 👍

Reply:
I may be missing something but I don't see them removed?

Reply:
`git push` helps 🤦‍♂️